### ISSUE TITLE
Follow Up AffineTransforms #648

### ISSUE THREAD

**Issue by vonnieda** [openpnp/openpnp#813]
These are a couple minor issues that need to be followed up on after the commit of #648.

* [x] Implement transform for the two point board locator.
* [ ] Add an indication to the user when a board has been located.
    See https://tips4java.wordpress.com/2010/01/24/table-row-rendering/
* [x] Verify compatibility with panels.
* [x] Implement inverted transform in calculateBoardPlacementLocationInverse so that manual training benefits from the transform.
* [x] Fix calculateBoardPlacementLocationInverse for board bottoms. Currently only works for top.
    I think the solution is just to apply the steps in reverse.

**Comment by vonnieda** [comment]
Some additional discussion in https://groups.google.com/d/msgid/openpnp/7a301f2e-7f61-4173-a841-8abe9d898b5d%40googlegroups.com?utm_medium=email&utm_source=footer

**Comment by vonnieda** [comment]
Note regarding "Fix calculateBoardPlacementLocationInverse for board bottoms. Currently only works for top.", I think the solution is just to apply the steps in reverse.

**Comment by markmaker** [comment]
@tonyluken, 

could you please look at this and tell me what you already implemented? So we can possibly close it?

Thanks!

_Mark

**Comment by tonyluken** [comment]
>Implement transform for the two point board locator.

I believe when I added support for more than three fiducials I also covered the two fiducial case as well so I think this is complete.

>Fix calculateBoardPlacementLocationInverse for board bottoms. Currently only works for top.
I think the solution is just to apply the steps in reverse.

I believe @vonnieda fixed that here [5efa1b0](https://github.com/openpnp/openpnp/commit/5efa1b0c16f6931b43d457aa6004565e9446886e)

>Add an indication to the user when a board has been located.

I think that may still be open.
